### PR TITLE
VMware Resource pool update to Ford Branch

### PR DESCRIPTION
**Pull request by sands6**
VMware resource pool feature set update. Terraform.tfvars have to updated accordingly for this change to work

### CODE REVIEW

**Comment by erasmus74** [comment]
Hitting EOF error trying to launch. Doing further testing now.

File: platforms/vmware/variables.tf [hunk 1]
```diff
@@ -114,6 +114,19 @@ variable "tectonic_vmware_etcd_clusters" {
 EOF
 }
 
+variable "tectonic_vmware_etcd_resource_pool" {
+  type = "map"
+
+  description = <<EOF
+  terraform map of etcd node(s) vSphere Resource Pools, example:
+  tectonic_vmware_etcd_clusters = {
```
Comment (metral) [inline platforms/vmware/variables.tf:122]: please update all examples to be _resource_pool not _clusters for: etcd, masters & workers

File: platforms/vmware/variables.tf [hunk 2]
```diff
@@ -231,6 +244,19 @@ variable "tectonic_vmware_master_clusters" {
 EOF
 }
 
+variable "tectonic_vmware_master_resource_pool" {
+  type = "map"
+
+  description = <<EOF
+  terraform map of master node(s) vSphere Resource pools, example:
+  tectonic_vmware_master_clusters = {
```
Comment (metral) [inline platforms/vmware/variables.tf:252]: same rename to _resource_pool as mentioned above

File: platforms/vmware/variables.tf [hunk 3]
```diff
@@ -343,6 +369,18 @@ variable "tectonic_vmware_worker_clusters" {
 EOF
 }
 
+variable "tectonic_vmware_worker_resource_pool" {
+  type = "map"
+
+  description = <<EOF
+  terraform map of worker node(s) vSphere Resource Pools, example:
+  tectonic_vmware_worker_clusters = {
```
Comment (metral) [inline platforms/vmware/variables.tf:377]: same rename to _resource_pool as mentioned above

**Comment by metral** [comment]
Please address feedback above and for all PRs, please do: `terraform fmt . ; make docs examples` to format all TF code, and regenerate any docs and examples that would've changed in the process.

**Comment by metral** [comment]
Please close this PR and open it against the 1.7.3 rebase branch after the edits are made: https://github.com/lander2k2/tectonic-installer/tree/1.7.3-tectonic.1-ford instead.

I've tested this PR in the lab on top of the new rebase, and it is working.

**Comment by lander2k2** [comment]
Closing.  Needs to be opened against the 1.7.3 branch.

**Pull request closed without merging**